src/output/workspace-impl.cpp

```diff
@@ -171,6 +171,10 @@ class output_layer_manager_t
         }
 
         view->damage();
+        for (auto ch : view->children)
```

Comment:
Uhh sadly not good enough, this won't work for children's children. Usually, we use `view->enumerate_views()` for such purposes, and it would also include the view itself .. And maybe we can separate this in its own function (albeit a short one)?

Reply:
yup was gonna ask about creating a function actually.

Reply:
Alright I created a small `damage_views()` fn.